spikingjelly/activation_based/cuda_kernel/spike_op.py

```
@@ -282,6 +406,12 @@ def spike_linear(
     if spike.get_device() < 0:
         return F.linear(spike, weight, bias)
     else:
+        if (
+            _CUSTOM_SPIKE_OP_READY
+            and use_cupy_custom_op()
+            and not isinstance(padding, str)
```

Ruff (0.15.12)

[error] Undefined name `padding`

(F821)